### PR TITLE
PE-7160: allow for boolean or string values of Allow-Unsafe-Addresses

### PR DESCRIPTION
**Pull request by arielmelendez**

### CODE REVIEW

**Review by fedellen — APPROVED** [review]
good catch!

File: spec/utils_spec.lua [hunk 1]
```diff
@@ -793,4 +793,32 @@ describe("utils", function()
...
+			assert.is_true(utils.booleanOrBooleanStringToBoolean("True"))
+			assert.is_true(utils.booleanOrBooleanStringToBoolean("TRUE"))
+			assert.is_true(utils.booleanOrBooleanStringToBoolean("tRuE"))
+			assert.is_false(utils.booleanOrBooleanStringToBoolean("false"))
+			assert.is_false(utils.booleanOrBooleanStringToBoolean("False"))
+			assert.is_false(utils.booleanOrBooleanStringToBoolean("FALSE"))
+			assert.is_false(utils.booleanOrBooleanStringToBoolean("fAlSe"))
+		end)
+
+		it("should not convert other truthy-like string values to boolean", function()
```
Comment (atticusofsparta) [inline spec/utils_spec.lua:814]: recommend adding a test for nil, table, function, and thread (coroutine)

**Review by atticusofsparta — DISMISSED** [review]

File: spec/utils_spec.lua (hunk 1 above)
Comment (arielmelendez) [reply spec/utils_spec.lua:814]: 51bbfe5
